packages/ember-simple-auth/addon/session-stores/cookie.js

```diff
@@ -173,11 +173,20 @@ export default BaseStore.extend({
 
   init() {
     this._super(...arguments);
-
     let owner = getOwner(this);
     if (owner && !this.hasOwnProperty('_fastboot')) {
       this._fastboot = owner.lookup('service:fastboot');
     }
+  },
+
+  _initialize(options) {
+    const clonedOptions = Object.assign({}, options);
+    if (clonedOptions.cookieName) {
+      this.set('_cookieName', clonedOptions.cookieName);
+    }
+
+    delete clonedOptions.cookieName;
+    this.setProperties(clonedOptions);
```

Comment:
Could this not remain in `init`? Having a special `_initialize` method makes things a bit more complex 🤔

Reply:
No because of this https://github.com/simplabs/ember-simple-auth/pull/2312#discussion_r675576773.

Basically tests had the luxury to add properties while creating the store, but now that the service is initialized by the container, and we still need to set values I've added `_initialize`.
Unfortunately this isn't something we could take care of with just `setProperties` because it'd invalidate a cookie with incorrect name which would make all tests fail.